### PR TITLE
Fixed issue #18986: superadmin without create superadmin can create superadmin

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: specific test for Permissions['superadmin']

### CODE REVIEW

**Comment by Shnoulle** [comment]
Done inside model function but not in this controller.

See https://github.com/LimeSurvey/LimeSurvey/blob/5c4fb4e0137d8e564c14d30c3c9563ffd7ecaacd/application/models/Permission.php#L296

**Review by gabrieljenik — APPROVED** [review]
Code looks good from structure point of view. Haven't analyzed the solution nor tested.

File: application/controllers/UserManagementController.php [hunk 1]
```diff
@@ -506,8 +506,10 @@ public function actionUserPermissions()
                 }
             }
             $aBasePermissions = $aFilteredPermissions;
+        } elseif(!Permission::model()->hasGlobalPermission('superadmin', 'create')) {
+            /* superadmin permission always need create */
```
Comment (gabrieljenik) [inline application/controllers/UserManagementController.php:510]: Why using multiline comments?
`//` isn't a better fit?

Comment (Shnoulle) [reply application/controllers/UserManagementController.php:510]: It's still an one line comment

Comment (Shnoulle) [reply application/controllers/UserManagementController.php:510]: I think i make a simple version for master : just unset the superadmin in arrau of Permission. I do same here.

I really think it more clear

**Pull request merged**